src/REstate.Engine.Repositories.Redis/REstate.Engine.Repositories.Redis.csproj

```diff
@@ -22,7 +22,6 @@
   <ItemGroup>
     <PackageReference Include="MessagePack" Version="1.7.3.4" />
     <PackageReference Include="StackExchange.Redis" Version="1.2.6" />
-    <PackageReference Include="System.Linq.Parallel" Version="4.3.0" />
```

Comment:
This was an unused package reference, it snuck into this PR, but I was going to remove it anyway.